### PR TITLE
Add winter hand-drawn landscape Fixes #462

### LINKED ISSUE #462: Add winter hand-drawn landscape
### Expected results
Winter hand-drawn landscape in game, with custom sound and animation.

### Actual results
-

### Steps to reproduce the issue
-

### Implementation/Solution Ideas
Draw winter-styled tiles and recreate it in Inkscape, create custom sound and cut it, add new tiles to the code, add animation.

### Would you like to work on this
Yes

### PR DESCRIPTION
**Pull request by kubami9**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

### Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link**
https://kubami9.github.io/labyrinth-km-contribute/

  Related Issues or Pull Requests: #462 

### The problem I want to solve or the facility I want to improve is/are
Add winter hand-drawn landscape with custom sound and simple animation

### My steps to solve it
- drawing images I wanted to use to create tiles
- recording custom sound
- importing drawings to Inkscape and editing it so it fits as a tile
- cutting sound
- including files in project folders
- adding code of winter tiles
- adding simple animation in one of the svg files

**Note:**
- tiles are fully drawn by me
- sound is also played by me on the piano, it's short begging of Jingle Bells (about 5/6 seconds)

If necessarily I can show my drawings on paper. Obviously I'm not a great artist but I did my best.

### CODE REVIEW

**Comment by niccokunzmann** [comment]
Hi, can you resolve conflicts? Also, please have a look at the documentation about adding music and add the music license.

**Comment by kubami9** [comment]
I think it's good now 😃 What do you think? @niccokunzmann

File: audio/music/jingleBellsKuba.mp3.license [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Sound by Kuba Michalski
+
+# Attribution 4.0 International (CC BY 4.0)
+By using this file you accept that it won't be used for destructive purposes.
```
Comment (niccokunzmann) [inline audio/music/jingleBellsKuba.mp3.license]: This additional line is problematic as I do not know what destructive means. This is in the license:

> No warranties are given. The license may not give you all of the permissions necessary for your intended use. For example, other rights such as publicity, privacy, or moral rights may limit how you use the material.

Please see if this covers your concerns.

**I can not guarantee when I hear your music, that no-one else uses it for destructive purposes.**

Comment (niccokunzmann) [reply audio/music/jingleBellsKuba.mp3.license]: Idea: please remove this line.

**Comment by niccokunzmann** [comment]
It is ready to merge. Is this your music? If yes, I can merge with this sentence. If not, please remove the sentence as you can not restrict the rights.

**Comment by niccokunzmann** [comment]
Okay, you can change that later if it is a problem. In short what I want to say is:

If you grant CC-BY, you can not restrict the use any further or it is not CC-BY. CC-BY + non-destructive purpose is not CC-BY any more.

**Comment by kubami9** [comment]
@niccokunzmann Okay, I'll keep this in mind and delete this line 😉 Yes, music is mine, recorded when playing piano :)

**Pull request merged**